charts/nexus/templates/proxy-route.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.nexusProxyRoute.enabled }}
+apiVersion: route.openshift.io/v1
+kind: Route
```

Comment:
What about removing these objects as we don't deploy on OpenShift and we will probably never do?